### PR TITLE
Migrate site to /scipy2014

### PR DESCRIPTION
**Pull request by mrterry**
Moves site root to /scipy2014
Splits static & media locations into separate directories so that collectstatic doesn't clobber our sponsor's images
Install libjpeg and libtiff to so that pillow & easy_thumbnail work

### CODE REVIEW

File: scipy2014/settings.py [hunk 1]
```diff
@@ -195,7 +195,7 @@
     "account.auth_backends.EmailAuthenticationBackend",
 ]
 
-LOGIN_URL = "/account/login/"  # @@@ any way this can be a url name?
+LOGIN_URL = "/scipy2014/account/login/"  # @@@ any way this can be a url name?
```
Comment (codersquid) [inline scipy2014/settings.py:198]: Not in 1.4. You can do it in 1.5 and on.

File: fabfile.py [hunk 2]
```diff
@@ -130,3 +132,9 @@ def deploy_mail(venv_path):
     render_to_file('django_mail_template.sh', 'django_mail.sh',
                    virtualenv=venv_path)
     scipy_put('django_mail.sh', pjoin(SITE_PATH, 'bin/django_mail.sh'))
+
+
+def install_system_deps():
```
Comment (codersquid) [inline fabfile.py:137]: Take a look at my fab_changes branch to see if you like the provisioning task that includes installing system dependencies. For apt-get stuff, I don't think you need to do it with every deploy, unless we blow the box away every time. So, one idea is to have the system-wide packages only done during provisionings. But the virtualenv stuff happens everytime.

Comment (mrterry) [inline fabfile.py:137]: right.  apt-get install is idempotent, so might as well make sure its is there.

Comment (codersquid) [inline fabfile.py:137]: oh, right then.

File: scipy2014/settings.py (hunk 1 above)
Comment (mrterry) [inline scipy2014/settings.py:198]: interesting.  i didn't know that.

**Comment by mrterry** [comment]
non_root2 does this better.

**Pull request closed without merging**